### DISCUSSION TITLE
How to install this code on HA Core

### DISCUSSION THREAD

**Discussion by Bolten88** [ksheumaker/homeassistant-apsystems_ecur#93] [Q&A]
Hello,

do you have an instruction on how to install this on HA Core?

Kind regards,
Mark Bolten

**Comment by HAEdwin** [comment]
Do you have HACS installed allready?

**Reply by Bolten88** [reply]
There is no add on store in Home Assistant core on Docker. So there is no HACS as far as I know.

**Reply by HAEdwin** [reply]
If you have little experience with HA it is best to have the front-end available allthough it is possible to use any of the three parts seperately and it might suit your needs it takes some reseach to get from A to B. Since this is probably not your only challenge I would advise to ask the question in the more generic forum because I personally have no experience with it if you only run HA Core. Maybe someone else who might read this?
